### PR TITLE
Add first-time cap docs

### PR DESCRIPTION
**Pull request by harimohanraj89**
**Story card:** [ch2820](https://app.clubhouse.io/simpledotorg/story/2820/test-terraform-scripting)

Adds documentation for first-time capistrano deployments, based on our learnings from doing a Kerala test run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -323,7 +323,41 @@ Rake tasks can be run on the deployed server using Capistrano as well. For examp
...
+```bash
+bundle exec cap <environment> sidekiq:install
+```
+
+#### 2. Deploy
+
+You can now run a regular Capistrano deployment:
+
+```bash
+bundle exec cap <environment> sidekiq:install
```
Comment (kpethtel) [inline README.md:355]: Is this a typo? It's the same as the sidekiq install above?

Comment (harimohanraj89) [reply README.md:355]: Yessir, that should be `deploy`.

Comment (kpethtel) [reply README.md:355]: Cool, I'm going to approve on the assumption that you'll correct this.

**Review by kpethtel — APPROVED** [review]
Thanks for doing this!

**Pull request merged**